tests/e2e/tests/e2e/OpenshiftConnector.spec.ts

```diff
@@ -76,13 +76,13 @@ suite('Openshift connector user story', async () => {
     await projectTree.waitProjectImported(projectName, 'index.js');
     await dashboard.waitDisappearanceNavigationMenu();
     await openshiftPlugin.clickOnOpenshiftToollBarIcon();
-    await openshiftPlugin.waitOpenshiftConnectorTree();
+    await openshiftPlugin.waitOpenshiftConnectorTree(10_000);
```

Comment:
Intentional?

Reply:
Also two more occurences of this in this file...

Reply:
Fixed, new variables created. At the end of the file, I left the 
```
await terminal.selectTabByPrefixAndWaitText('OpenShift: Push', 'Changes successfully pushed to component', 30_000);
```
as the default timeout for this method (3 000ms) is overwritten.